src/redux/features/auth.slice.ts

```diff
@@ -0,0 +1,44 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+export interface AuthState {
+  modalMode: "login" | "sign-up" | "forgot-psw" | null;
+  showModal: boolean;
+}
+
+const initialState: AuthState = {
+  modalMode: null,
+  showModal: false
+};
+
+export const authSlice = createSlice({
+  name: "auth",
+  initialState,
+  reducers: {
+    showLoginFormModal: (state: AuthState) => {
```

Comment:
({ ...state, modalMode: "login" , showModal : true })

Reply:
I have tried doing like that but it have type error. Can you fix for me?